src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -716,7 +716,8 @@ private void SetRequestTracingOptions()
                 IsDevEnvironment = TracingUtils.IsDevEnvironment(),
                 IsKeyVaultConfigured = _options.IsKeyVaultConfigured,
                 IsKeyVaultRefreshConfigured = _options.IsKeyVaultRefreshConfigured,
-                FeatureManagementSchemaVersion = _options.FeatureManagementSchemaVersion
+                FeatureManagementSchemaVersion = _options.FeatureManagementSchemaVersion,
+                ReplicaCount = _options.Endpoints?.Count() ?? 1,

```

Comment:
When you say `ReplicaCount`, does that mean number of replicas or number of endpoints? If its number of replicas, should it be number of endpoints -1?

Reply:
Good point. In the current implementation, it is number of endpoints. I considered using `EndpointsCount` but didn't go that way because of space constraints. But I like the suggestion of just reducing `1` from the `endpoints` to get replicas.

Reply:
Done!